Makefile

```diff
@@ -2,7 +2,11 @@ PG_MAJOR?=12
 # All PG_VERSIONS binaries/libraries will be included in the Dockerfile
 # specifying multiple versions will allow things like pg_upgrade etc to work.
 PG_VERSIONS?=12 11
+
+# Additional PostgreSQL extensions we want to include with specific version/commit tags
 POSTGIS_VERSIONS?="2.5 3"
+PG_AUTH_MON?=a38b2341
```

Comment:
Bit awkward perhaps, but `master` being a moving target, I'd rather point it to a know commit.